sound/soc/sof/topology.c

```diff
@@ -2348,23 +2348,29 @@ static int sof_dspless_widget_ready(struct snd_soc_component *scomp, int index,
 	if (WIDGET_IS_DAI(w->id)) {
 		struct snd_sof_dev *sdev = snd_soc_component_get_drvdata(scomp);
 		struct snd_sof_widget *swidget;
-		struct snd_sof_dai dai;
+		struct snd_sof_dai *sdai;
```

Comment:
Minor correction for the commit message:
`This minor change should not have any impact on existing DAIs, even
when the DSP is` s/supported/used ?

Reply:
done.